modules/ROOT/pages/silverblue-guide.adoc

```diff
@@ -159,6 +159,136 @@ there have likely noted when they did a remote-ls command on their configured
 flathub remote, there are more things listed than the app's. There are runtimes,
 and sdk's as well as app's since Flathub.org is a place to distribute such things.
 
+==== Installing Flatpak Applications
+===== Gnome Software
+Gnome Software is the default graphical software installation application that
```

Comment:
Please capitalize GNOME, throughout.

Reply:
Okay. I changed all cases of Gnome to GNOME.